lib/bios/menus.robot

```diff
@@ -663,10 +663,29 @@ Get Firmware Version From Tianocore Setup Menu
 Get USB Boot Option
     [Documentation]    Returns the full name of the first boot option
     ...    containing "USB"
-    ${menu_construction}=    Enter Boot Menu Tianocore And Return Construction
-    FOR    ${option}    IN    @{menu_construction}
-        ${match}=    Run Keyword And Return Status    Should Match Regexp
-        ...    ${option}    .*[Uu][Ss][Bb].*
-        IF    ${match}    RETURN    ${option}
+    ${menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${menu}
+    ...    One Time Boot
+    Press Key N Times    1    ${ARROW_DOWN}
+    Read From Terminal
+    Press Key N Times    1    ${ARROW_UP}
```

Comment:
After entering submenu, first `Read From Terminal` would return string with `Device Path:` but without footer:

```text
Return: '                   |\n| LCtrl+LAlt+F12=Save screenshot                                               |\n\\------------------------------------------------------------------------------/^v=Move Highlight       <Enter>=Select Entry\n\n\n\n                                                         Device Path :\n                                                         PciRoot(0x0)/Pci(0x1F,\n                                                         0x2)/Sata(0x0,0xFFFF,0\n                                                         x0)/HD(1,GPT,C719B99E-\n                                                         A21F-4544-94FF-6BA1F34\n                                                         61BC4,0x800,0x100000)/\n                                                         \\EFI\\ubuntu\\shimx64.ef\n                                                         i'
```

After that on each `ARROW_DOWN`, `Read From Terminal` returns:

```
Return: '\n\n\n\n   Ubuntu (on QEMU HARDDISK)                             Device Path :\n   QEMU HARDDISK                                         PciRoot(0x0)/Pci(0x1F,\n                                                         0x2)/Sata(0x0,0xFFFF,0\n                                                         x0)\n\n\n\n\n\n\n\n\n\n\n\n\n                          F9=Reset to Defaults      F10=Save\n  ^v=Move Highlight       <Enter>=Select Entry      Esc=Exit\n  LCtrl+LAlt+F12=Save screenshot'
```

So it's either:
* go down, clear buffer and go back up again
* somehow rework loop so that first iteration reads from `Device Path :` to the end

Should I add comments explaining how it works?

Reply:
Or in more readable form something like this:
First Read From Terminal:
```text
| LCtrl+LAlt+F12=Save screenshot                                               |
\\------------------------------------------------------------------------------/
^v=Move Highlight       <Enter>=Select Entry



                                                         Device Path :
                                                         PciRoot(0x0)/Pci(0x1F,
                                                         0x2)/Sata(0x0,0xFFFF,0
                                                         x0)/HD(1,GPT,C719B99E-
                                                         A21F-4544-94FF-6BA1F34
                                                         61BC4,0x800,0x100000)/
                                                         \\EFI\\ubuntu\\shimx64.ef
                                                         i
```

Second one:
```
   Ubuntu (on QEMU HARDDISK)                             Device Path :
   QEMU HARDDISK                                         PciRoot(0x0)/Pci(0x1F,
                                                         0x2)/Sata(0x0,0xFFFF,0
                                                         x0)












                          F9=Reset to Defaults      F10=Save
  ^v=Move Highlight       <Enter>=Select Entry      Esc=Exit
  LCtrl+LAlt+F12=Save screenshot
```